scene/3d/lightmap_gi.cpp

```diff
@@ -1086,6 +1056,69 @@ LightmapGI::BakeError LightmapGI::bake(Node *p_from_node, String p_image_data_pa
 		return BAKE_ERROR_MESHES_INVALID;
 	}
 
+	// POSTBAKE: Save Textures.
+
+	TypedArray<TextureLayered> textures;
+	{
+		Vector<Ref<Image>> images;
+		images.resize(lightmapper->get_bake_texture_count());
+		for (int i = 0; i < images.size(); i++) {
+			images.set(i, lightmapper->get_bake_texture(i));
+		}
+
+		int slice_count = images.size();
+		int slice_width = images[0]->get_width();
+		int slice_height = images[0]->get_height();
+
+		int slices_per_texture = Image::MAX_HEIGHT / slice_height;
+		int texture_count = Math::ceil(slice_count / (float)slices_per_texture);
+
+		textures.resize(texture_count);
+		images.resize(texture_count);
```

Comment:
I think this is the cause of the error, this shouldn't be resized, right?

Reply:
Probably. I don't remember how this line was added :/